api/src/exam-environment/routes/exam-environment.test.ts

```diff
@@ -524,15 +526,43 @@ describe('/exam-environment/', () => {
     });
 
     xdescribe('POST /exam-environment/screenshot', () => {});
+
+    describe('GET /exam-environment/exams', () => {
+      it('should return 200', async () => {
+        const res = await superGet('/exam-environment/exams').set(
+          'exam-environment-authorization-token',
+          examEnvironmentAuthorizationToken
+        );
+        expect(res.status).toBe(200);
+
+        expect(res.body).toMatchObject({
```

Comment:
```suggestion
        expect(res.body).toStrictEqual({
```
Nit, strict is better, if possible.